app/workers/rapid_ready_for_decision/form526_asthma_job.rb

```diff
@@ -32,7 +32,17 @@ def query_and_assess_lighthouse(form526_submission)
     def assess_medications(medications)
       return [] if medications.blank?
 
-      RapidReadyForDecision::LighthouseMedicationRequestData.new(medications).transform
+      asthma_keywords = RapidReadyForDecision::Constants::DISABILITIES[:asthma][:keywords]
+      transformed_medications = RapidReadyForDecision::LighthouseMedicationRequestData.new(medications).transform
+      flagged_medications = transformed_medications.map do |medication|
+        {
+          **medication,
+          flagged: asthma_keywords.any? do |keyword|
+            medication.to_s.downcase.include?(keyword)
+          end
```

Comment:
This could be a one-liner:
```suggestion
          flagged: asthma_keywords.any? { |keyword| medication.to_s.downcase.include?(keyword) }
```

Reply:
I agree.